### PR TITLE
Added note about namespace for Knative Serving

### PR DESCRIPTION
**Pull request by abrennan89**

### CODE REVIEW

**Comment by alanfx** [comment]
Looks good to me, @abrennan89 !

File: modules/serverless-installing-knative-serving.adoc [hunk 1]
```diff
@@ -7,6 +7,11 @@
 
 You must create a `KnativeServing` object to install Knative Serving using the {ServerlessOperatorName}.
 
+[IMPORTANT]
+====
+The `KnativeServing` object must be created in the `knative-serving` namespace as shown in the sample yaml, otherwise the object will be ignored.
```
Comment (kalexand-rh) [inline modules/serverless-installing-knative-serving.adoc]: I'd probably say:

> You must create the `KnativeServing` object in the `knative-serving` namespace, as shown in the sample YAML, or it is ignored.

s/yaml/YAML in either case

**Comment by abrennan89** [comment]
Thanks @kalexand-rh - should be ready now :slightly_smiling_face:

**Pull request merged**